Source/Core/Primitives/Math/MathF.cs

```diff
@@ -1246,6 +1246,34 @@ public static void Swap<T>(ref T first, ref T second)
 		/// <param name="baseHash"></param>
 		/// <param name="otherHash"></param>
 		/// <returns></returns>
+		public static void CombineHashCodeUint(ref uint baseHash, uint otherHash)
```

Comment:
- I think you don't need `uint` versions of `CombineHashCode` if you just use the `int` versions and do a cast to `uint` inside an `unchecked` block.
- Please remove `uint` overloads to keep maintenance cost / gain ratio lower.